css/card.css

```diff
@@ -1,29 +1,85 @@
-
-.card, .card-front, .card-back {
-    display: inline;
+.card,
+.card-front,
+.card-back {
+  display: inline;
 }
 
 /* Override default button styling */
-.card-front, .card-back {
-    display: inline;
-    width: 200px;
-    margin: 3%;
-    border: 0;
-    padding: 0;
-    background: none;
-    cursor: pointer;
+.card-front,
+.card-back {
+  display: inline;
+  width: 200px;
+  margin: 3%;
+  border: 0;
+  padding: 0;
+  background: none;
+  cursor: pointer;
 }
 
-.card-image, .card-cover {
-    width: 100%;
+.card-image,
+.card-cover {
+  width: 100%;
 }
 
 .card-cover {
-    margin: 3%;
-    border: solid;
-    border-color: dodgerblue;
+  margin: 3%;
+  border: solid;
+  border-color: dodgerblue;
+}
+
+.dialog-title {
```

Comment:
Dialogs and cards are intertwined, but might be worth putting dialog in a separate file or making this file named card-and-dialog

Will decide later after merging in o5v3's upcoming file reorg